spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/ModuleDeploymentWriter.java

```diff
@@ -86,32 +77,19 @@
 	/**
 	 * Amount of time to wait for a status to be written to all module
 	 * deployment request paths.
-	 */
-	private final long timeout;
-
-
-	/**
-	 * Construct a {@code ModuleDeploymentWriter} with the default timeout
-	 * value indicated by {@link #DEFAULT_TIMEOUT}.
 	 *
-	 * @param zkConnection         ZooKeeper connection
-	 * @param containerMatcher     matcher for modules to containers
 	 */
-	public ModuleDeploymentWriter(ZooKeeperConnection zkConnection, ContainerMatcher containerMatcher) {
-		this(zkConnection, containerMatcher, DEFAULT_TIMEOUT);
-	}
+	private final AtomicLong deploymentTimeout;
```

Comment:
Do we expect this value to change during runtime? If not, I think using `final long` instead of `AtomicLong` should suffice here.